frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts

```diff
@@ -51,6 +57,30 @@ const isBeingMigrated = (vm: VMKind, migrations?: K8sResourceKind[]): VMStatus =
   return NOT_HANDLED;
 };
 
+const isBeingStopped = (vm: VMKind, launcherPod: PodKind = null): VMStatus => {
+  if (isVMReady(vm) || isVMCreated(vm)) {
+    const podStatus = getPodStatus(launcherPod);
+    const ContainerStatuses = getPodContainerStatuses(launcherPod);
+    if (ContainerStatuses) {
```

Comment:
Nit: we can reduce cyclomatic complexity

```ts
if (!ContainerStatuses) return NOT_HANDLED;

// ..
```

Reply:
Done.